test/functional/shared/resources/aws_s3_bucket_test.go

```diff
@@ -129,5 +129,6 @@ func Test_AWS_S3Bucket_Existing(t *testing.T) {
 		},
 	})
 
+	test.RequiredFeatures = []shared.RequiredFeature{shared.FeatureAWS}
```

Comment:
Similar changes will be made in subsequent PRs once the tests are separated out.